src/app/courses/view-courses/courses-view.component.ts

```diff
@@ -86,9 +86,9 @@ export class CoursesViewComponent implements OnInit, OnDestroy {
   }
 
   resourceUrl(stepDetail) {
-    if (Object.keys(stepDetail.attachment.doc._attachments)[0]) {
```

Comment:
@paulbert , @lmmrssa  It only checked for the first item. What is the plan for the multiple resources ? Please suggest me.